index.js

```diff
@@ -69,7 +80,7 @@ module.exports = {
     publicAssets.forEach((assetType) => {
       trees.push(
         new Funnel(
-          `${this.app.bowerDirectory}/upfluence-oss/${assetType}/`,
+          `node_modules/upfluence-oss-components/node_modules/upfluence-oss/${assetType}/`,
```

Comment:
in the final project, upfluence-oss will be not installed at the root of the project node_module dir ?

Reply:
nope, the only way to do that is to have `upfluence-oss` as a direct dependency of the final project :/

Reply:
hmm it's wierd because the most of packages in the node_module are a direct dependency of the final projet